R/course.R

```diff
@@ -328,6 +331,19 @@ parse_content_disposition <- function(cd) {
   )
 }
 
+progress_fun <- function(down, up) {
```

Comment:
The default curl progress bar gets mangled in (some versions of?) RStudio, so I took @jeroen's advice to copy an approach from tesseract.

https://github.com/ropensci/tesseract/blob/6514c5862a6e6991c45048449b20fd24babf4bb4/R/tessdata.R#L63-L74